lib/pf/Switch.pm

```diff
@@ -1817,23 +1817,6 @@ sub reverseBitmask {
     return $flippedBitMask;
 }
 
-=item getSysUptime - returns the sysUpTime
-
-=cut
-
-sub getSysUptime {
```

Comment:
It is not used anywhere. Removed because I originally reworked it but since we are no longer automatically invalidating the cache i removed both (the original and the rework)